inference-engine/src/gna_plugin/layers/gna_layer_info.hpp

```diff
@@ -90,6 +90,30 @@ class LayerInfo {
         static InferenceEngine::details::caseless_set<std::string> layersWithConstrains = {"memory", "convolution"};
         return layersWithConstrains.find(name) != layersWithConstrains.end();
     }
+    size_t getOutputBatchSize() const {
+        if (!layer) {
+            THROW_GNA_EXCEPTION << "layer is null";
+        }
+        if (!layer->outData[0]) {
+            THROW_GNA_EXCEPTION << "output data of layer '" << layer->name << "' is null";
+        }
+        auto& dims = layer->outData[0]->getDims();
+        auto layout = layer->outData[0]->getLayout();
+        switch (dims.size()) {
```

Comment:
dims size can be equal to 1, in this case batch size will be 1, please, add this case, otherwise 1d tensors support will be broken

Reply:
done